src/Sentry/Protocol/Context/Contexts.cs

```diff
@@ -78,5 +83,57 @@ internal void CopyTo(Contexts to)
                 to.TryAdd(kv.Key, value);
             }
         }
+
+        /// <inheritdoc />
+        public void WriteTo(Utf8JsonWriter writer)
+        {
+            writer.WriteDictionaryValue(this!);
```

Comment:
Curious why does it need `!`?

Reply:
No idea :/

Reply:
![image](https://user-images.githubusercontent.com/1935960/100636801-c7e93000-333a-11eb-9fd0-7e0abb529445.png)

No idea. The receiving type is even more accepting. I think nullability covariance is just buggy in C#.